ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -64,19 +65,25 @@
   private final boolean activeBlobsOnly;
   // True if blob stats needs to be logged, false otherwise
   private final boolean logBlobStats;
+  // set to true if only error logging is required
+  private final boolean silent;
```

Comment:
Also if someone wanted only errors wouldn't they just set the relevant log level in log4j props?